### PR TITLE
Add command to sort student

### PR DESCRIPTION
**Pull request by mahadhir247**
Fixes issue #94

### CODE REVIEW

**Review by whelan-low — APPROVED** [review]
LGTM! Though I'm thinking if it would be good to add a test case to make sure the sorting order is accurate, done with a small number of students.

File: src/main/java/seedu/address/logic/commands/SortStudentCommand.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    private final String sortBy;
+    public SortStudentCommand(String sortBy) {
+        this.sortBy = sortBy;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        Comparator<Person> personComparator;
+        switch (sortBy) {
```
Comment (Jajared) [inline src/main/java/seedu/address/logic/commands/SortStudentCommand.java:40]: Good handling of input edge cases here by converting to lower case in the parser. You can consider using Enum types instead of string for determining what to sort by to make it more robust.

File: src/test/java/seedu/address/logic/parser/SortStudentCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    private SortStudentCommandParser parser = new SortStudentCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, SortStudentCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validArgs_returnsSortStudentCommand() {
```
Comment (Jajared) [inline src/test/java/seedu/address/logic/parser/SortStudentCommandParserTest.java:28]: Since you have handled the edge case in the parser, it would be nice if you could also show different user inputs such as "Name", "NAME", "name" (etc) to show that we covered all cases

Comment (Jajared) [reply src/test/java/seedu/address/logic/parser/SortStudentCommandParserTest.java:28]: Nevertheless the current test cases are more than sufficient! Good job 👍
